src/triggers/ipfs.ts

```diff
@@ -28,3 +28,35 @@ export const unpinnedTrackContent: (cidField: string, limit?: number) => Trigger
 
     return _.uniq(cids);
   };
+
+export const audioNotOnIpfs: Trigger<undefined> = async (clients: Clients) => {
+  const query = `select t.* from "${Table.processedTracks}" as t
+      left outer join "${Table.ipfsFiles}" i
+      on t."lossyAudioURL" = i.url
+      where "lossyAudioIPFSHash" is null
+      and "lossyAudioURL" is not null
+      and i.error is null
+      LIMIT ${process.env.QUERY_TRIGGER_BATCH_SIZE!}`
+
+  const tracks = (await clients.db.rawSQL(
+    query
+  )).rows
+
+  return tracks
```

Comment:
this is with the join, so maybe tracksWithFiles ?

Reply:
we join on files to only get the tracks without file upload errors, but we only select the columns from tracks so this is still really just tracks